save-frontend/src/main/kotlin/com/saveourtool/save/frontend/routing/BasicRouting.kt

```diff
@@ -188,7 +188,7 @@ val basicRouting: FC<AppProps> = FC { props ->
                 smallText = "Page not found"
                 withRouterLink = true
             } to ERROR_404,
-
+            banView.create { userInfo = props.userInfo } to BAN,
```

Comment:
But where is a redirect for users to this page in case they were banned? 
And same about user api - we need to prevent usage if the user was banned.

Reply:
The redirect is in app.kt
The user api will be configured in the next pr.